cmd/snap/cmd_quota.go

```diff
@@ -45,24 +42,45 @@ The quotas command shows all quota groups.
 `)
 
 var shortRemoveQuotaHelp = i18n.G("Remove quota group")
-var longRemoveQuotaHelp = i18n.G(`The remove-quota command removes the given quota group.`)
+var longRemoveQuotaHelp = i18n.G(`
+The remove-quota command removes the given quota group.
 
-type cmdQuota struct {
-	clientMixin
-	// MaxMemory and MemoryMax are mutually exclusive and provided for
-	// convienience, they mean the same.
-	MaxMemory  string `long:"max-memory" optional:"true"`
-	MemoryMax  string `long:"memory-max" optional:"true"`
-	Parent     string `long:"parent" optional:"true"`
-	Positional struct {
-		GroupName string              `positional-arg-name:"<group-name>" required:"true"`
-		Snaps     []installedSnapName `positional-arg-name:"<snap>" optional:"true"`
-	} `positional-args:"yes"`
-}
+Quota groups with sub-groups 
+`)
+
+var shortSetQuotaHelp = i18n.G(`Create or update a quota group.`)
+var longSetQuotaHelp = i18n.G(`
+The set-quota command updates or creates a quota group with the specified set of
+snaps.
+
+A quota group sets resource limits (currently maximum memory only) on the set of
+snaps that belong to it. Snaps can be at most in one quota group. Quota groups
+can be nested.
+
+All snaps provided are appended to the group; to remove a snap from a
+quota group the entire group must be removed with remove-quota and recreated 
+without the quota group. To remove a sub-group from the quota group, the 
+sub-group must be removed directly with the remove-quota command.
+
+The memory limit for a quota group can be increased, but cannot be decreased. To
+decrease the memory limit for a quota group, the entire group must be removed
+with the remove-quota command and recreated with the lower limit. Increasing the
+memory limit for a quota group does not restart any services associated with 
+snaps in the quota group.
+
+Adding new snaps to a quota group will result in all non-disabled services in 
+that snap being restarted.
```

Comment:
is this true, we don't modify the slice itself to add snaps to it?

Reply:
Yes, this is true, the slice does not need to be modified at all when we add it, I mean internally when we do the systemctl daemon-reload systemd will move processes into it and probably does some accounting stuff, but the slice file does not need to be modified on disk, and the slice does not need to be restarted